src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        FigureSupplier figureSupplier = new FigureSupplier(random, colorSupplier);
+        for (int i = 0; i < 5; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        for (int i = 5; i < 6; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
```

Comment:
The first half of figures in this array should be generated with random parameters. From requirements. So change a little bit the conditions of your loops.